blocks-config/forms/class-uagb-forms.php

```diff
@@ -190,6 +190,24 @@ public function process_forms() {
 
 			$post_content = get_post_field( 'post_content', sanitize_text_field( $_POST['post_id'] ) );
 
+			if ( defined( 'ASTRA_ADVANCED_HOOKS_POST_TYPE' ) ) {
+				$option = array(
+					'location'  => 'ast-advanced-hook-location',
+					'exclusion' => 'ast-advanced-hook-exclusion',
+					'users'     => 'ast-advanced-hook-users',
+				);
+				$result = Astra_Target_Rules_Fields::get_instance()->get_posts_by_conditions( ASTRA_ADVANCED_HOOKS_POST_TYPE, $option );
+
+				if ( ! empty( $result ) || is_array( $result ) ) {
```

Comment:
should it be && here
```suggestion
				if ( ! empty( $result ) && is_array( $result ) ) {
```
We want not empty array, right?

Reply:
Done